tests/failpoints/mod.rs

```diff
@@ -9,10 +9,13 @@ extern crate slog_global;
 
 mod cases;
 
+use std::sync::Once;
+
+static INIT: Once = Once::new();
+
 fn setup<'a>() -> fail::FailScenario<'a> {
```

Comment:
Could you call `let _ = setup();` between line 23 and 24? So we can test whether multiple setups lead to panic.

Reply:
It will cause deadlock because `fail::FailScenario::setup()` acquires a lock and releases it when drops.

Reply:
Oh, sorry. It will be dropped before the next `setup()`.